plugins/feeds/public/CertPaIT.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from datetime import datetime, timedelta
+from core.observables import Hash
+from core.feed import Feed
+from core.errors import ObservableValidationError
+
+class CertPaIt(Feed):
+
+    default_values = {
+        "frequency": timedelta(minutes=30),
+        "name": "CertPaIT",
+        "source" : "https://infosec.cert-pa.it/analyze/submission.rss",
+        "description": "This feed contains data from infosec.cert-pa.it",
+    }
+
+    def update(self):
+        for item in self.update_xml('item', ["title", "link"]):
+            self.analyze(item)
+
+    def analyze(self, item):
+        md5 = item['title'].replace("MD5: ", "")
+        context = {}
+        context['date_added'] = datetime.now()
```

Comment:
You might want to use the feeds `pubDate` here, as using `now()` will create a new context entry every time the feed runs.

Reply:
i checked published_parsed and it was None, but in default csv reader it worked, retesting